Formula/eye-d3.rb

```diff
@@ -3,15 +3,14 @@ class EyeD3 < Formula
 
   desc "Work with ID3 metadata in .mp3 files"
   homepage "https://eyed3.nicfit.net/"
-  url "https://eyed3.nicfit.net/releases/eyeD3-0.9.5.tar.gz"
-  mirror "https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.5.tar.gz"
-  sha256 "faf5806197f2093e82c2830d41f2378f07b3a9da07a16fafb14fc6fbdebac50a"
-  license "GPL-3.0"
-  revision 2
+  url "https://eyed3.nicfit.net/releases/eyeD3-0.9.6.tar.gz"
+  mirror "https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.6.tar.gz"
```

Comment:
```suggestion
  url "https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.6.tar.gz"
  mirror "https://eyed3.nicfit.net/releases/eyeD3-0.9.6.tar.gz"
```

Reply:
I already tried this. It caused this error:
```
==> brew audit eye-d3 --online --git --skip-style
==> FAILED
eye-d3:
  * Stable: The URL https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.6.tar.gz is not reachable (HTTP status code 404)
Error: 1 problem in 1 formula detected
```

Reply:
CI run logs: https://github.com/Homebrew/homebrew-core/runs/1628308614